Demo2.js

```diff
@@ -3,6 +3,12 @@
 import React from 'react';
 import Springs from './Springs';
 
+if (!String.prototype.includes) {
+  String.prototype.includes = function() {'use strict';
+    return String.prototype.indexOf.apply(this, arguments) !== -1;
+  };
+}
+
```

Comment:
Oh sorry, seems I was mistaken. I was testing on chrome mainly and babel-node and thought it was auto enabled. Guess we should just enable the option instead?